### PR TITLE
Mostly changes to the test text to make them more accurate.

### PR DESCRIPTION
**Pull request by tchaffee**
#### Pre-Submission Checklist

- [x] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file
- [x] Working changes have been added to the build by running `npm run build`

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [ ] Tested changes locally.

- [ ] Closes currently open issue: Closes #XXX

#### Description

Improved the log of test 14, so please pay extra attention to the changes there.
All the other changes should just be changes to the test text to more accurately describe what is actually being tested.

### CODE REVIEW

File: build/bundle.js [hunk 1]
```diff
@@ -20455,24 +20455,24 @@ var FCC_Global =
...
 
-	      it('9. If I enter numbers outside the range of the number field, I will\n      see an HTML5 validation error.\'', function () {
+	      it('9. If I enter numbers outside the range of the number input, I will\n      see an HTML5 validation error.\'', function () {
 	        var number = document.getElementById('number');
 	        FCC_Global.assert.isNotNaN(parseInt(number.min), 'Minimum number should be defined ');
 	        FCC_Global.assert.isNotNaN(parseInt(number.max), 'Maximum number should be defined ');
 	      });
 
-	      it('10. For the name, email, and number input fields, I can see\n      corresponding labels that describe the purpose of each field with the\n      following ids: id="name-label", id="email-label", and id="number-label".', function () {
+	      it('10. For the name, email, and number input fields inside the form I can\n      corresponding labels that describe the purpose of each field with the\n      following ids: id="name-label", id="email-label", and id="number-label".', function () {
```
Comment (tbushman) [inline build/bundle.js]: Please re-insert 'see'.

File: build/bundle.js [hunk 2]
```diff
@@ -20532,24 +20532,19 @@ var FCC_Global =
...
-	        var checkboxes = document.querySelectorAll('input[type="checkbox"]');
-	        // need to make sure there are more than two checkboxes, as our default
-	        // code contains a checkbox as well if we are detecting only two, one of
-	        // them is ours; our checkbox does not have a value-attribute so we
-	        // must use "-1" in the third test
-	        FCC_Global.assert.isAtLeast(checkboxes.length, 3, 'There should be at least 2 checkboxes ');
-	        FCC_Global.assert.isAtLeast(document.querySelectorAll('#survey-form input[type="checkbox"]').length, 2, 'The checkboxes are not inside the form element ');
-	        FCC_Global.assert.strictEqual(checkboxes.length - 1, getNumValues(checkboxes), 'All your checkboxes must have a value attribute ');
+	      it('14. Inside the form element, I can select several fields from a\n      series of checkboxes, each of which must have a value attribute.', function () {
+	        var checkboxes = document.querySelectorAll('#survey-form input[type="checkbox"]');
```
Comment (tbushman) [inline build/bundle.js]: Good solution.

File: build/bundle.js (hunk 1 above)
Comment (tchaffee) [inline build/bundle.js]: Good catch, thanks!

**Comment by tchaffee** [comment]
I made the requested change, should be ready for merge now.

**Pull request merged**